runtime/src/snapshot_utils.rs

```diff
@@ -1904,6 +1907,7 @@ pub fn package_and_archive_full_snapshot(
     snapshot_version: SnapshotVersion,
     maximum_full_snapshot_archives_to_retain: usize,
     maximum_incremental_snapshot_archives_to_retain: usize,
+    accounts_hash: Hash,
```

Comment:
Can this field be removed? From a quick glance it looks like this fn param is always called with `bank.accounts_hash()` anyway.